### PR TITLE
DO-123: Refactored AppHeaderMenuProfile.jsx

### PR DESCRIPTION
**Pull request by mohitsaladiya**

### CODE REVIEW

File: src/components/common/AppHeader/AppHeaderDesktop/AppHeaderDesktopMenu/AppHeaderMenuProfile/AppHeaderProfileContent/AppHeaderProfileDropdown/AppHeaderProfileDropdownList/AppHeaderProfileDropdownList.jsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import AppHeaderProfileDropdownListItem from './AppHeaderProfileDropdownListItem';
+import { Routes } from '../../../../../../../../../constants/common/commonConstants';
+
+/**
+ * Renders AppHeaderProfileDropdownList
+ * @return {Component}
+ * @constructor
+ */
+const AppHeaderProfileDropdownList = () => (
+  <ul>
```
Comment (ayushmalik2110) [inline src/components/common/AppHeader/AppHeaderDesktop/AppHeaderDesktopMenu/AppHeaderMenuProfile/AppHeaderProfileContent/AppHeaderProfileDropdown/AppHeaderProfileDropdownList/AppHeaderProfileDropdownList.jsx:14]: Do we need to create an un-order list component ?

File: src/components/common/AppHeader/AppHeaderDesktop/AppHeaderDesktopMenu/AppHeaderMenuProfile/AppHeaderProfileContent/AppHeaderProfileDropdown/AppHeaderProfileDropdownList/AppHeaderProfileDropdownListItem/AppHeaderProfileDropdownListItem.jsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+/**
+ * Renders App Header Profile Dropdown list item
+ * @param {Object} props
+ * @return {Component}
+ */
+const AppHeaderProfileDropdownListItem = (props) => {
+  const { path, icon, text } = props;
+  return (
+    <li>
```
Comment (ayushmalik2110) [inline src/components/common/AppHeader/AppHeaderDesktop/AppHeaderDesktopMenu/AppHeaderMenuProfile/AppHeaderProfileContent/AppHeaderProfileDropdown/AppHeaderProfileDropdownList/AppHeaderProfileDropdownListItem/AppHeaderProfileDropdownListItem.jsx:16]: And a list component too.

File: src/components/common/AppHeader/AppHeaderDesktop/AppHeaderDesktopMenu/AppHeaderMenuProfile/AppHeaderProfileContent/AppHeaderProfileDropdown/AppHeaderProfileDropdownList/AppHeaderProfileDropdownList.jsx (hunk 1 above)
Comment (mohitsaladiya) [reply src/components/common/AppHeader/AppHeaderDesktop/AppHeaderDesktopMenu/AppHeaderMenuProfile/AppHeaderProfileContent/AppHeaderProfileDropdown/AppHeaderProfileDropdownList/AppHeaderProfileDropdownList.jsx:14]: I totally agree that we should have a typography component for this.

File: src/components/common/AppHeader/AppHeaderDesktop/AppHeaderDesktopMenu/AppHeaderMenuProfile/AppHeaderProfileContent/AppHeaderProfileDropdown/AppHeaderProfileDropdownList/AppHeaderProfileDropdownListItem/AppHeaderProfileDropdownListItem.jsx (hunk 2 above)
Comment (mohitsaladiya) [reply src/components/common/AppHeader/AppHeaderDesktop/AppHeaderDesktopMenu/AppHeaderMenuProfile/AppHeaderProfileContent/AppHeaderProfileDropdown/AppHeaderProfileDropdownList/AppHeaderProfileDropdownListItem/AppHeaderProfileDropdownListItem.jsx:16]: Agreed.

File: src/components/common/AppHeader/AppHeaderDesktop/AppHeaderDesktopMenu/AppHeaderMenuProfile/AppHeaderProfileContent/AppHeaderProfileDropdown/AppHeaderProfileDropdownList/AppHeaderProfileDropdownList.jsx (hunk 1 above)
Comment (mohitsaladiya) [reply src/components/common/AppHeader/AppHeaderDesktop/AppHeaderDesktopMenu/AppHeaderMenuProfile/AppHeaderProfileContent/AppHeaderProfileDropdown/AppHeaderProfileDropdownList/AppHeaderProfileDropdownList.jsx]: Done

File: src/components/common/AppHeader/AppHeaderDesktop/AppHeaderDesktopMenu/AppHeaderMenuProfile/AppHeaderProfileContent/AppHeaderProfileDropdown/AppHeaderProfileDropdownList/AppHeaderProfileDropdownListItem/AppHeaderProfileDropdownListItem.jsx (hunk 2 above)
Comment (mohitsaladiya) [reply src/components/common/AppHeader/AppHeaderDesktop/AppHeaderDesktopMenu/AppHeaderMenuProfile/AppHeaderProfileContent/AppHeaderProfileDropdown/AppHeaderProfileDropdownList/AppHeaderProfileDropdownListItem/AppHeaderProfileDropdownListItem.jsx]: Done

**Pull request merged**